<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>process</class>
 <widget class="QDialog" name="process">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>164</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>164</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>164</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Illumination</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <property name="spacing">
    <number>12</number>
   </property>
   <item>
    <widget class="QFrame" name="form_area">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="layout">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="layout_2">
          <item>
           <widget class="QLabel" name="final_temperature_label">
            <property name="text">
             <string>Final Temperature (°C)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="final_temperature">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <double>600.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_3">
          <item>
           <widget class="QLabel" name="heating_rate_label">
            <property name="text">
             <string>Heating Rate (°C/s)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="heating_rate">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="minimum">
             <double>0.100000000000000</double>
            </property>
            <property name="maximum">
             <double>20.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_4">
          <item>
           <widget class="QLabel" name="stabilization_label">
            <property name="text">
             <string>Stabilization (s)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="stabilization">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <double>99999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontal_spacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontal_spacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layout_5">
        <property name="spacing">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="layout_6">
          <item>
           <widget class="QLabel" name="optical_power_label">
            <property name="text">
             <string>Optical Power (%)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="optical_power">
            <property name="minimumSize">
             <size>
              <width>55</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>55</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_7">
          <item>
           <widget class="QLabel" name="ligth_source_label">
            <property name="text">
             <string>Ligth Source</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ligth_source">
            <property name="minimumSize">
             <size>
              <width>58</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>58</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Blue</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IR</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>AUX</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_8">
          <item>
           <widget class="QLabel" name="time_label">
            <property name="text">
             <string>Time (s)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="time">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <double>99999.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="button_area">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="push_button_accept">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Accept</string>
        </property>
        <property name="shortcut">
         <string notr="true">Return</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="push_button_cancel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="shortcut">
         <string notr="true">Esc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="push_button_information">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Information</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
